Cargo.lock

```diff
@@ -1260,25 +1334,48 @@ dependencies = [
  "tokio-rustls",
 ]
 
+[[package]]
+name = "iana-time-zone"
+version = "0.1.53"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "64c122667b287044802d6ce17ee2ddf13207ed924c712de9a66a5814d5b64765"
+dependencies = [
```

Comment:
It's crazy that the timezone database package "needs" a javascript runtime, a WASM binding, the winapi binding, the Android binding, and the Mac OS binding.  Let's find a way to remove this.